types/mouse-event-offset/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "esModuleInterop": true,
```

Comment:
This should never be enabled in DefinitelyTyped

Reply:
How do I have a default import without this? @andrewbranch